AsyncDisplayKit/ASControlNode.m

```diff
@@ -254,6 +269,7 @@ - (void)addTarget:(id)target action:(SEL)action forControlEvents:(ASControlNodeE
       [targetActions addObject:NSStringFromSelector(action)];
     });
 
+  _targetAdded = YES;
```

Comment:
We would need to clear this in the remove target method, right? In that case, does it make sense to instead remove the state and inspect _controlEventDispatchTable, potentially in an accessory method like -(BOOL)hasTarget ?